static/js/scanners/scannerrule.js

```diff
@@ -0,0 +1,35 @@
+const selectedScannerIsYara = (select) => {
```

Comment:
@diox do we have unit tests for such js files?

Reply:
We don't :(